packages/form-js-editor/src/features/properties-panel/groups/ValidationGroup.js

```diff
@@ -9,16 +9,33 @@ import {
   isNumberFieldEntryEdited,
   isTextFieldEntryEdited,
   NumberFieldEntry,
-  TextFieldEntry
+  TextFieldEntry,
+  SelectEntry
 } from '@bpmn-io/properties-panel';
 
 import { useService } from '../hooks';
 
 import { INPUTS } from '../Util';
 
+const VALIDATION_TYPE_OPTIONS = {
+  custom: {
+    value: 'custom',
+    label: 'Custom',
+  },
+  email: {
+    value: 'email',
+    label: 'Email',
+  },
+  phone: {
+    value: 'phone',
+    label: 'Phone',
+  },
+};
 
 export default function ValidationGroup(field, editField) {
   const { type } = field;
+  const validate = get(field, [ 'validate' ], {});
+  const isCustomValidation = [ undefined, VALIDATION_TYPE_OPTIONS.custom.value ].includes(validate?.validationType);
```

Comment:
Just to raise alternatives: instead of changing the schema by adding `validationType` we could detect the type by the pattern. But I guess this is too much magic, so having the `validationType` is the most explicit way 👍